centreon/www/include/configuration/configObject/host/DB-Func.php

```diff
@@ -1019,7 +1020,11 @@ function insertHostInDB($ret = array(), $onDemandMacro = null)
     updateNagiosServerRelation($hostId, $ret);
 
     $ret = $form->getSubmitValues();
-    if (isset($ret['dupSvTplAssoc']['dupSvTplAssoc']) && $ret['dupSvTplAssoc']['dupSvTplAssoc']) {
+
+    if (
+        (isset($ret['dupSvTplAssoc']['dupSvTplAssoc']) && $ret['dupSvTplAssoc']['dupSvTplAssoc'])
```

Comment:
Idea: does 100% the same but is more concise.
```suggestion
        (! empty($ret['dupSvTplAssoc']['dupSvTplAssoc'])
```

Reply:
Ok thanks 👍🏻